lib/filter.g

```diff
@@ -171,6 +171,60 @@ BIND_GLOBAL( "InstallTrueMethodNewFilter", function ( tofilt, from )
 end );
 
 
+#############################################################################
+##
+#F SuspendMethodReordering()
+#F ResumeMethodReordering()
+#F ResetMethodReordering()
+##
+##  <#GAPDoc Label="MethodReordering">
+##  <ManSection>
+##  <Func Name="SuspendMethodReordering" Arg=""/>
+##  <Func Name="ResumeMethodReordering" Arg=""/>
+##  <Func Name="ResetMethodReordering" Arg=""/>
+##
+##  <Description> These functions control whether the method reordering process
+##  described in <Ref Func="InstallTrueMethod"/> is invoked or not. Since this 
+##  process can be comparatively time-consuming, it is usually suspended when
+##  a lot of implications are due to be installed, for instance when loading
+##  the library, or a package. This is done by called <C>SuspendMethodReordering()</C>
```

Comment:
called -> calling? Also, the recommend usage is that *every* reference to a function should use `<Ref Func=...>`, even if it is inside the body of the documentation of the function itself.